lib/bunch/math.ex

```diff
@@ -43,4 +43,21 @@ defmodule Bunch.Math do
   def div_rem(dividend, divisor, accumulated_remainder) do
     div_rem(accumulated_remainder + dividend, divisor)
   end
+
+  @doc """
+  Returns `value` decreased by `value (mod divisor)`
+
+  ## Examples
+
+      iex> #{inspect(__MODULE__)}.int_part(10, 4)
+      8
+      iex> #{inspect(__MODULE__)}.int_part(7, 7)
+      7
+
+  """
+  @spec int_part(value :: non_neg_integer, divisor :: pos_integer) :: non_neg_integer
```

Comment:
We've been discussing names for 2 variants of this function. The best I came up with is: `min_multiple_gt` and `max_multiple_lt`. If you think they're good enough, you can implement them, if not - this function really needs a better name.